packages/maskbook/src/plugins/ITO/UI/ClaimGuide.tsx

```diff
@@ -36,10 +37,13 @@ interface ClaimGuideProps extends Pick<ClaimDialogProps, 'exchangeTokens' | 'pay
 export function ClaimGuide(props: ClaimGuideProps) {
     const { t } = useI18N()
     const { payload, exchangeTokens, isBuyer, open, retryPayload, onClose } = props
+    const [startTransition] = unstable_useTransition({ busyDelayMs: 1000 })
```

Comment:
IIRC `busyDelayMs` is removed in react 👀

Reply:
It is in our codebase.